scripts/arabica.sh

```diff
@@ -57,4 +57,4 @@ echo "Downloading genesis file..."
 celestia-appd download-genesis ${CHAIN_ID}
 
 echo "Starting celestia-appd..."
-celestia-appd start --v2-upgrade-height 1751707 --force-no-bbr
+celestia-appd start --v2-upgrade-height 1751707
```

Comment:
I think this is still useful to have because even if your OS supports BBR, not necessarily you want it to be enabled. Similar to other scripts too.

What do you think @rootulp?

Reply:
i'll defer to your decision on this